### PR TITLE
Fix: Remove the `system-Exception` ignore errors.

### PR DESCRIPTION
**Pull request by kai890707**
**Description** 
See #7731
 
 **Checklist:**
 
 * [x]  Securely signed commits
 * [ ]  Component(s) with PHPDoc blocks, only if necessary or adds value
 * [ ]  Unit testing, with >80% coverage
 * [ ]  User guide updated
 * [x]  Conforms to style guide

### CODE REVIEW

**Review by MGatner — CHANGES_REQUESTED** [review]

File: system/Exceptions/CastException.php [hunk 1]
```diff
@@ -27,6 +27,9 @@ public function getExitCode(): int
         return EXIT_CONFIG;
     }
 
+    /**
+     * @return CastException
```
[the remark this answers is not in the document]
Comment (kai890707) [reply system/Exceptions/CastException.php]: Thank you, I have edited it.

File: system/Exceptions/TestException.php [hunk 2]
```diff
@@ -18,6 +18,9 @@ class TestException extends CriticalError
 {
     use DebugTraceableTrait;
 
+    /**
+     * @return TestException
```
Comment (paulbalandan) [inline system/Exceptions/TestException.php:22]:
```suggestion
     * @return static
```

**Review by datamweb — APPROVED** [review]

**Review by paulbalandan — APPROVED** [review]

**Comment by kenjis** [comment]
Please rebase to resolve conflicts.
See https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/workflow.md#updating-your-branch

**Comment by kai890707** [comment]
> Please rebase to resolve conflicts. See https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/workflow.md#updating-your-branch

I synchronized the current develop branch and ran phpstan.

**Comment by kenjis** [comment]
> I synchronized the current develop branch and ran phpstan.

What is *synchronized* ? What did you do exactly? 

This branch is broken. See the diffs:
https://github.com/codeigniter4/CodeIgniter4/pull/7791/files

**Comment by kenjis** [comment]
If you don't know how to use git, I recommend you try https://learngitbranching.js.org/

**Pull request closed without merging**